babel.config.js

```diff
@@ -9,6 +9,7 @@ module.exports = function generateConfig(api) {
             browsers: ['last 2 version', 'ie >= 11'],
           },
           useBuiltIns: 'usage',
+          // modules: false,
```

Comment:
```suggestion

```

Reply:
this was a typo and is different in final solution .